### PR TITLE
Adding deploy tutorial for Aerobatic

### PR DESCRIPTION
**Pull request by dvonlehman**

### CODE REVIEW

File: docs/docs/deploy-gatsby.md [hunk 1]
```diff
@@ -291,3 +292,42 @@ not caused by Gatsby. React uses HTML comments to help identify locations of
...
+  # Note with below setting it is not neccessary to pass --directory to aero deploy command
+  directory: public
+  # Turn off the Aerobatic asset fingerprinting since Gatsby already does this
+  optimizer:
+    fingerprintAssets: false
+
+plugins:
+  # Force aggressive 1yr max-age header for all .js and .js.map requests
+  - name: http-headers
+    path: ['/*.js', '/*.js.map']
```
Comment (KyleAMathews) [inline docs/docs/deploy-gatsby.md]: This won't be safe until gatsby v2 actually https://www.gatsbyjs.org/docs/caching/

Comment (dvonlehman) [reply docs/docs/deploy-gatsby.md]: Oh I see. I'll update the PR to instruct that only the headers only be set to cache forever for the `/static/*` pattern.

Comment (KyleAMathews) [reply docs/docs/deploy-gatsby.md]: Would love to get this in when you have a chance to push the update!

**Comment by m-allanson** [comment]
Thanks @dvonlehman! I've updated your PR to merge into the version 2 branch of Gatsby, so no further changes are needed. It means your changes won't be published until the Gatsby site switches over to v2. You can [check out the v2 project to see how it's coming along](http://github.com/gatsbyjs/gatsby/projects/2).

**Comment by dvonlehman** [comment]
@m-allanson that's great! Sorry I let this sit around idle for so long.

**Comment by m-allanson** [comment]
No problem, thanks for the PR!

**Comment by jlengstorf** [comment]
Hiya @dvonlehman! 👋

This is definitely late, but on behalf of the entire Gatsby community, I wanted to say thank you for being here.

Gatsby is built by awesome people like you. Let us say “thanks” in two ways:

1.  **We’d like to send you some Gatsby swag.** As a token of our appreciation, you can go to the [Gatsby Swag Store][store] and log in with your GitHub account to get a coupon code good for one free piece of swag. (We’ve got t-shirts and hats, plus some socks that are really razzing our berries right now.)
2.  **If you’re not already part of it, we just invited you to join the Gatsby organization on GitHub.** This will add you to our team of maintainers. You’ll receive an email shortly asking you to confirm. By joining the team, you’ll be able to label issues, review pull requests, and merge approved pull requests.

If you have questions, please don’t hesitate to reach out to us: tweet at [@gatsbyjs][twitter] and we’ll come a-runnin’.

Thanks again! 💪💜

[store]: https://store.gatsbyjs.org
[twitter]: https://twitter.com/gatsbyjs

**Pull request merged**